auth/authentication.js

```diff
@@ -2,12 +2,14 @@ const db = require('../models');
 const logger = require('winston');
 
 module.exports = {
-  authenticateChannelCredentials (channelName, userPassword) {
+  authenticateChannelCredentials (skipAuth, channelName, userPassword) {
```

Comment:
Why call this then if `skipAuth` is true?

Reply:
good point.  I was struggling with how to best do this in the promise chain, but I now abstracted this to its own function